requirements/dev-requirements.txt

```diff
@@ -14,5 +14,6 @@ pylint==2.12.2
 pyroma==3.2
 pytest-cov==3.0.0  # Measure code coverage
 pytest==6.2.5
+sphinx-prompt>=0.1
```

Comment:
I think that this is best as a requirement in a `prompt` (install extra), so our users can `pip install sphinx-subsititution-extensions[prompt]`.

I'm happy to discuss this.

Reply:
Sounds good! Today I learned about python extras 😄